test/LiveServer.hs

```diff
@@ -339,7 +339,76 @@ liveServerUnitTests = testGroup "Unit tests against a live server"
       step "Validate content-type"
       liftIO $ assertEqual "Content-Type did not match" (Just "application/javascript") (Map.lookup "Content-Type" m)
       step "Cleanup actions"
+
+      removeObject bucket object
+
+  , funTestWithBucket "putObject contentLanguage tests" $ \step bucket -> do
+      step "fPutObject content language test"
+      let object = "xxx-content-language"
+          size1 = 100 :: Int64
+
+      step "create server object with content-language"
+      inputFile <- mkRandFile size1
+      fPutObject bucket object inputFile def{
+        pooContentLanguage = Just "en-US"
+        }
+
+      -- retrieve obj info to check
+      oi <- headObject bucket object
+      let m = oiMetadata oi
+
+      step "Validate content-language"
+      liftIO $ assertEqual "content-language did not match" (Just "en-US") (Map.lookup "content-language" m)
+      step "Cleanup actions"
+
+      removeObject bucket object
+
+  , funTestWithBucket "putObject storageClass tests" $ \step bucket -> do
+      step "fPutObject storage class test"
+      let object = "xxx-storage-class-standard"
+          object' = "xxx-storage-class-reduced"
+          object'' = "xxx-storage-class-invalid"
+          size1 = 100 :: Int64
+          size0 = 0 :: Int64
+
+      step "create server objects with storageClass"
+      inputFile <- mkRandFile size1
+      inputFile' <- mkRandFile size1
+      inputFile'' <- mkRandFile size0
+
+      fPutObject bucket object inputFile def{
+        pooStorageClass = Just "STANDARD"
+        }
+
+      fPutObject bucket object' inputFile' def{
+        pooStorageClass = Just "REDUCED_REDUNDANCY"
+        }
+
```

Comment:
Server won't return "STANDARD" storage class in response header during `headObject`, so either we can modify the test or remove it and retain only "REDUCED_REDUNDANCY" test.

Reply:
fixed